test/known_issues/test-child-process-exec-stdout-data-string.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/7342
+const common = require('../common');
+const assert = require('assert');
+const exec = require('child_process').exec;
+
+const keepAlive = setInterval(() => {}, 9999);
+
+const command = common.isWindows ? 'dir' : 'ls';
+const e = exec(command);
```

Comment:
Could you name this something like `child`? `e` seems like it would be an error.